src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/AssessmentServiceTest.kt

```diff
@@ -125,30 +122,34 @@ class AssessmentServiceTest {
 
       assertThat(currentAssessment.assessmentEntity).isEqualTo(anOffender.currentAssessment())
 
-      val eligibilityProgress = currentAssessment.eligibilityProgress()
+      val eligibilityProgress = currentAssessment.getEligibilityProgress()
       assertThat(eligibilityProgress).hasSize(11)
       assertThat(eligibilityProgress).allMatch { it.status == NOT_STARTED }
 
-      val suitabilityProgress = currentAssessment.suitabilityProgress()
+      val suitabilityProgress = currentAssessment.getSuitabilityProgress()
       assertThat(suitabilityProgress).hasSize(7)
       assertThat(suitabilityProgress).allMatch { it.status == SuitabilityStatus.NOT_STARTED }
     }
 
     @Test
     fun `for an offender with some progress`() {
-      val offender = anOffenderWithSomeProgress()
+      val assessment = anAssessmentWithSomeProgress(
+        ELIGIBILITY_AND_SUITABILITY_IN_PROGRESS,
+        eligibilityProgress = Progress.specifyByIndex(0 to PASSED),
+        suitabilityProgress = Progress.specifyByIndex(0 to PASSED),
```

Comment:
Added some helpers to allow fine grain tweaking of progress levels - this might be a bit odd to read but it basically means assume that they passed the first eligibility criterion and first suitability criterion